javascript-practice/src/js/views/commonLoginRegisterView.js

```diff
@@ -0,0 +1,117 @@
+import CommonView from './commonView';
+import * as CONSTANT from '../constants/constant';
+import { validatePassword } from '../helpers/helpers';
+
+export default class CommonLoginRegisterView extends CommonView {
+  constructor() {
+    super();
+
+    this.parentElement = document.querySelector('.form');
+    this.messageDefault = CONSTANT.MESSAGE.ERROR_MESSAGE_DEFAULT;
+    this.inputField = document.querySelectorAll('.form__input');
+  }
+
+  /**
+   * Validate user input data
+   * @param {Object} account The account object with email, password, passwordConfirm field
+   * @returns {boolean} Return true if validate success and return false if validate not success
+   */
+  validateForm(account) {
+    if (account.password === account.passwordConfirm) {
+      if (validatePassword(account.passwordConfirm)) {
+        return true;
+      }
+      this.showError(CONSTANT.MESSAGE.PASSWORD_NOT_STRONG);
+      return false;
+    }
+    this.showError(CONSTANT.MESSAGE.PASSWORD_NOT_MATCH);
+    return false;
+  }
+
+  /**
+   * Add event listener for form input
+   * @param {Function} handler The function need to be set event
+   */
+  addHandlerForm(handler) {
+    this.parentElement.addEventListener('submit', (e) => {
+      e.preventDefault();
+      this.clearErrorMessage();
+      handler();
+    });
+  }
+
+  /**
+   * Show error style input password
+   */
+  changeToStyleErrorInputPassword() {
```

Comment:
Make your function name is more simple, pls

Reply:
Fixed.